provider/cmd/pulumi-resource-purrl/schema.json

```diff
@@ -31,18 +33,21 @@
         "nodejs": {
             "dependencies": {
                 "@pulumi/pulumi": "^3.0.0"
-            }
+            },
+            "packageName": "@pulumiverse/pulumi-purrl"
         },
         "python": {
+            "packageName": "pulumiverse_pulumi_purrl",
             "requires": {
                 "pulumi": ">=3.0.0,<4.0.0"
             }
         }
     },
     "license": "Apache-2.0",
     "name": "purrl",
+    "pluginDownloadURL": "github://api.github.com/pulumiverse/pulumi-purrl",
```

Comment:
It's enough pointing to the organization: `github://api.github.com/pulumiverse`
Because your provider is named `purrl`, Pulumi will automatically search in repo `pulumi-purrl` of our organization.

Reply:
done!